app/repositories/SessionRepository.scala

```diff
@@ -54,14 +55,14 @@ class SessionRepository @Inject()(
 
   private def byId(id: String): Bson = Filters.equal("_id", id)
 
-  def keepAlive(id: String): Future[Boolean] =
+  def keepAlive(id: String): Future[Unit] =
     collection
       .updateOne(
         filter = byId(id),
         update = Updates.set("lastUpdated", Instant.now(clock)),
       )
       .toFuture
-      .map(_ => true)
+      .as(())
```

Comment:
minor: do you think it's worth adding an extra function to FutureOps to handle this case? (like .unit in cats)